pkgs/development/libraries/jemalloc/default.nix

```diff
@@ -52,6 +52,10 @@ stdenv.mkDerivation rec {
 
   preConfigure =
     ''
+      # we don't have access to the git commit for GID here, see
+      # https://github.com/NixOS/nixpkgs/issues/8567#issuecomment-2398540893
+      GID=$(basename $out | cut -d '-' -f 1)
+      echo "${version}-0-g$GID" > VERSION
```

Comment:
The `configure` script takes a `--with-version` argument; let’s pass that instead as a more public interface.

Also, if we’re going to use a fake commit hash, let’s just do `${version}-0-g0000000000000000000000000000000000000000` rather than confusing people with a real‐seeming hash that isn’t a Git object.

However, jemalloc doesn’t exactly get releases very often, so another option would be to simply define the commit hash too and keep it updated manually. Or fetch the tarball as well and get the `VERSION` file from that.

Reply:
> Also, if we’re going to use a fake commit hash, let’s just do `${version}-0-g0000000000000000000000000000000000000000` rather than confusing people with a real‐seeming hash that isn’t a Git object.

that works for me, updated!